src/main/java/uk/gov/mca/beacons/api/utils/OffsetDateTimeOptionalZoneDeserializer.java

```diff
@@ -0,0 +1,21 @@
+package uk.gov.mca.beacons.api.utils;
+
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.databind.DeserializationContext;
+import com.fasterxml.jackson.databind.deser.std.StdScalarDeserializer;
+import java.io.IOException;
+import java.time.OffsetDateTime;
+
+public class OffsetDateTimeOptionalZoneDeserializer
```

Comment:
Is this class used at all? It doesn't appear to be in the code?

Reply:
I mentioned in a comment that I had meant to use it with BeaconDTO, it's safe to remove if you didn't use it. https://github.com/mcagov/beacons-service/pull/156#issuecomment-915854748